cms/server/admin/handlers/base.py

```diff
@@ -69,6 +70,13 @@
 logger = logging.getLogger(__name__)
 
 
+def natural_sort_key(text):
```

Comment:
I think it was better to sort more simply, we can always improve it later (and if we decide to actually use natural sorting I would prefer to do it through a library that handles things like accent normalization etc)